pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -593,6 +606,9 @@ func (o *RoleModificationOptions) AddRole() error {
 				}
 			case rbacv1.GroupKind:
 				if o.UserClient != nil {
+					if _, ok := systemGroups[newSubject.Name]; ok {
```

Comment:
You used a `map[...]bool` so the succinct way to check is simply `if map[key] { ... }`.